client/components/LearnerViewDirective.js

```diff
@@ -296,9 +296,9 @@ tie.directive('learnerView', [function() {
 
 
         $scope.questionIds = questionSet.getQuestionIds();
-        $scope.unlockedQuestions = [];
+        $scope.completedQuestions = [];
         for (var i = 0; i < $scope.questionIds.length; i++) {
-          $scope.unlockedQuestions.push(false);
+          $scope.completedQuestions.push(false);
```

Comment:
Per discussion with @eyurko, renaming to `questionsCompletionStatus` instead so the code is more self-documenting.